.circleci/continue_config.yml

```diff
@@ -0,0 +1,248 @@
+  golang-promotion-test:
+    shell: /bin/bash -eo pipefail
+    executor:
+      name: go/default # is based on cimg/go
+      tag: '1.21'
+    steps:
+      - checkout
+      - install-just
+      - install-foundry
+      - run:
+          name: run validation checks
+          command: just promotion-test
+      - slack/notify:
+          event: always
+          custom: "{\n  \"blocks\": [\n    {\n      \"type\": \"section\",\n      \"text\": {\n        \"type\": \"mrkdwn\",\n        \"text\": \":control_knobs: The daily report on standard candidate chains is ready!\"\n      }\n    },\n    {\n      \"type\": \"divider\"\n    },\n    {\n      \"type\": \"context\",\n      \"elements\": [\n        {\n          \"type\": \"mrkdwn\",\n          \"text\": \"\U0001F440 Click the link to see what validation checks would fail if all candidate chains were promoted to standard (and exclusions removed).\"\n        }\n      ]\n    }\n  ]\n}\n"
+          channel: C07GZVCCUS0 # to slack channel `notify-superchain-promotion-failures`
+  publish-bot:
+    environment:
+      NODE_AUTH_TOKEN: $NPM_TOKEN # Use NPM_TOKEN as the auth token
+    docker:
+      - image: cimg/node:18 # Use Node.js 18
+    steps:
+      - checkout
+      - run:
+          name: Set deployment token
+          command: npm config set '//registry.npmjs.org/:_authToken' "${NPM_TOKEN}"
+      - env:
+          NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+      - run:
+          name: "Build and publish package on NPM \U0001F4E6"
+          command: pnpm release
+  check-codegen:
+    executor:
+      name: go/default # is based on cimg/go
+      tag: '1.21'
+    steps:
+      - checkout
+      - install-just
+      - run:
+          name: Run codegen
+          command: just codegen
+      - run:
+          name: check git tree is clean
+          command: git diff --exit-code
+  golang-validate-genesis-allocs:
+    docker:
+      - image: cimg/go:1.22.6-node
+    resource_class: medium
+    parameters:
+      chainid:
+        type: string
+    steps:
+      - checkout
+      - install-pnpm
+      - install-just
+      - install-foundry
+      - install-gvm
+      - install-nvm
+      - run: just validate-genesis-allocs <<parameters.chainid>>
+workflows:
+  hourly:
+    jobs:
+      - golang-validate-all:
+          context:
+            - slack
+      - golang-test:
+          context:
+            - slack
+    triggers:
+      - schedule:
+          cron: "0 * * * *"
+          filters:
+            branches:
+              only:
+                - main
+  nightly:
+    jobs:
+      - golang-promotion-test:
+          context:
+            - slack
+    triggers:
+      - schedule:
+          cron: "0 0 * * *"
+          filters:
+            branches:
+              only:
+                - main
+  pr-checks:
+    jobs:
+      - golang-validate-genesis-allocs:
+          matrix:
+            parameters:
+              # This value will be replaced by the generate_test_config.sh script
```

Comment:
Is below the fallback list, if `targetList` is empty in `generate_test_config.sh`?